src/mono/sample/wasm/browser-es6/Wasm.Browser.Sample.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename this to `Wasm.Browser.ES6.Sample.csproj`, else it will just conflict with the existing browser sample project, and only one of them would get picked up for testing.

` Zipping directory "/__w/1/s/src/mono/sample/wasm/browser-es6/bin/Release/AppBundle/" to "/__w/1/s/src/mono/sample/wasm/browser-es6/bin/browser-wasm/Wasm.Browser.Sample.zip".`
`Zipping directory "/__w/1/s/src/mono/sample/wasm/browser/bin/Release/AppBundle/" to "/__w/1/s/src/mono/sample/wasm/browser/bin/browser-wasm/Wasm.Browser.Sample.zip".`